flixel/addons/weapon/FlxWeapon.hx

```diff
@@ -479,7 +513,7 @@ class FlxTypedWeapon<TBullet:FlxBullet>
 
 enum FlxWeaponFireFrom
 {
-	PARENT(parent:FlxSprite, offset:FlxBounds<FlxPoint>);
+	PARENT(parent:FlxSprite, offset:FlxBounds<FlxPoint>, ?useParentAngle:Bool);
```

Comment:
Can you replace `?useParentAngle:Bool` with `useParentAngle:Bool = false`? There's no good reason to turn this into a `Null<Bool>` (see http://stackoverflow.com/questions/31914614/is-question-mark-operator-in-haxe-really-necessary/31916466#31916466).